### ISSUE TITLE
Ability to generate docs to path other than /jsondoc/

### ISSUE THREAD

**Issue by DougBraidwood** [fabiomaffioletti/jsondoc#209]
JSONDoc is serving up the documentation at the endpoint /jsondoc/jsondoc-ui.html

I would like to map it somewhere else e.g. /lowerlevel/jsondoc/jsondoc-ui.html

And when I look at the [code in GitHub](https://github.com/fabiomaffioletti/jsondoc/blob/master/jsondoc-springmvc/src/main/java/org/jsondoc/springmvc/controller/JSONDocController.java) I think this is coming from line 26 of JSONDocController

`public final static String JSONDOC_DEFAULT_PATH = "/jsondoc";`

It would be great if we could have a way to override the JSONDOC_DEFAULT_PATH

**Comment by ST-DDT** [comment]
AFAICT this is a limitation of spring.
Since it is used in the annotation you cannot overwrite it anyway.
However you can overwrite it if you subclass the controller.

**Comment by joffrey-bion** [comment]
According to [this stackoverflow answer](https://stackoverflow.com/a/41254553/1540818), it seems that it could be configurable via properties.

**Comment by ST-DDT** [comment]
This can be changed by doing the following:

````java
@Controller
public class CustomJSONDocController extends JSONDocController {

	public final static String JSONDOC_CUSTOM_PATH = "/apidoc";

	public CustomJSONDocController(String version, String basePath, List<String> packages) {
		super(version, basePath, packages);
	}

	@RequestMapping(value = JSONDOC_CUSTOM_PATH, method = RequestMethod.GET, produces = MediaType.APPLICATION_JSON_VALUE)
	public @ResponseBody JSONDoc getApi() {
		return super.getAPI();
	}

}
````
Just make sure you don't have the other class on your component scanning.

**Comment by joffrey-bion** [comment]
@ST-DDT Yes, I meant it could be *made* configurable. My comment's goal was to provide an idea as to how to implement this feature.
